tests/e2e/cucumber/environment/index.ts

```diff
@@ -68,11 +68,14 @@ Before(async function (this: World, { pickle }: ITestCaseHookParameter) {
     }
   })
   if (!config.basicAuth) {
-    await setAdminToken(state.browser)
-  }
-
-  if (config.keycloak) {
-    await setKeycloakAdminToken(state.browser)
+    // Currently, access token are received for keycloak via login
```

Comment:
```suggestion
    // Currently, access token are received from keycloak via login
```
from maybe or ?

Reply:
This note is just for to-do task that I am doing currently